wgpu-core/src/command/mod.rs

```diff
@@ -79,7 +79,7 @@ impl<A: hal::Api> CommandEncoder<A> {
         }
     }
 
-    fn open(&mut self) -> &mut A::CommandEncoder {
+    pub fn open(&mut self) -> &mut A::CommandEncoder {
```

Comment:
Did I need to make this method public, and CommandBuffer.encoder pub(crate)? Not sure.